pkgs/development/libraries/science/math/mkl/default.nix

```diff
@@ -51,18 +58,22 @@ stdenvNoCC.mkDerivation rec {
       cp license.txt $out/lib/
   '';
 
+  # fixDarwinDylibName fails for libmkl_cdft_core.dylib because the
+  # larger updated load commands do not fit. Use install_name_tool
+  # explicitly and ignore the error.
+  postFixup = stdenvNoCC.lib.optionalString stdenvNoCC.isDarwin ''
+      for f in $out/lib/*.dylib; do
+          install_name_tool -id $out/lib/`basename $f` $f || true
```

Comment:
Nitpick: I'd use `$()` instead of the backticks to stay clean as far as shell linters argue:
https://github.com/koalaman/shellcheck/wiki/SC2006

Reply:
Fixed, thanks.